src/neo4j_graphrag/neo4j_queries.py

```diff
@@ -209,5 +209,5 @@ def get_query_tail(
         return retrieval_query
     if return_properties:
         return_properties_cypher = ", ".join([f".{prop}" for prop in return_properties])
-        return f"RETURN node {{{return_properties_cypher}}} as node, score"
+        return f"RETURN node {{{return_properties_cypher}}} AS node, labels(node) AS nodeLabels, elementId(node) AS id, score"
```

Comment:
A wider point not directly related to this PR, but in the future do we want to come up with some rules for when we use 'id' across the package. Sometimes we use it to refer to elementId and sometimes we use it to refer to the ID created by the user or by the KG creation pipeline.

Reply:
Good point